packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -81,8 +86,59 @@
             border-radius: 0.5rem;
         }
     }
+
+    &__card-content-over-header {
+        display: flex;
+        background: #dfeaff;
```

Comment:
is there any existing colour for this? if not maybe you could create the new colour tone in constant.scss